### DISCUSSION TITLE
How to start learning and developing symbolic maths application

### DISCUSSION THREAD

**Discussion by nedzadimamovic10** [sympy/sympy#24155] [General]
I am trying to develop an interactive programme which will teach young students basic algebra. I have no experience in symbolic maths programming and I do not know even where to start from in learning how to develop a python code which can handle expressions symbolically. Can anybody suggest some literature and/or example from which I would be able to learn how to design and develop my application. My main interest at present is to enter algebraic expressions and then solve them, simplify etc. 
Kind regards, 
Nedzad

**Comment by oscarbenjamin** [comment]
You might get more responses if you ask this on the mailing list rathe than here.

**Comment by nedzadimamovic10** [comment]
Where is the mailing list? Can you direct me please.

Kind regards,

Nedzad
________________________________

**Reply by oscarbenjamin** [reply]
https://groups.google.com/g/sympy
